app/controllers/activity_groups_controller.rb

```diff
@@ -7,7 +7,7 @@ class ActivityGroupsController < ApplicationController
 
   # GET /activity_groups
   def index
-    @activity_groups = ActivityGroup.list.order(:name).page params[:page]
+    @activity_groups = ActivityGroup.list_group.order(:name).page params[:page]
```

Comment:
```suggestion
    @activity_groups = ActivityGroup.group_list.order(:name).page params[:page]
```
Solo porque lo hemos puesto así en otros modelos.